lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -255,3 +256,49 @@ def test_flag_disabled(self, mock_logger):
             'For this mobile request, user activity is not enabled for this user {} and course {}'.format(
                 str(self.user.id), str(self.course.id))
         )
+
+
+@ddt.ddt
+class TestBlocksInfoInCourseView(TestBlocksInCourseView):  # lint-amnesty, pylint: disable=test-inherits-tests
```

Comment:
Add test cases for all the response types mentioned in response status codes returned by this view.

Reply:
Just because we are extending TestBlocksInCourseView we decided and agreed to focus only on testing the changes in response (new fields as `media`, `course dates`, `certificate`)  exist in response